get_models_and_test.py

```diff
@@ -119,17 +121,12 @@ def mean_of_values_at_indices(arr, indices):
                             cdf_counter = 0
                             
                             
-                            multi_lstm_model = toy_models.get_fitted_model(data_input=data_config, 
-                                                                            model_name=model_name, 
-                                                                            epochs=epochs, 
-                                                                            force_retrain = force_retrain_models, 
-                                                                            lstm_units=lstm_size,
-                                                                            qth=0.5)
+                            dqn_lstm = DQNLSTM(qth,epochs=epochs,data_config=data_config,model_name=model_name,lstm_units=lstm_size)
```

Comment:
Is it possible to put DQNLSTM inside "toy_models"? That way, this line shouldn't need changing. Instead, we can just add a string "dqn" (or similar) to the `model_prefix_names` array

Reply:
Ok, will try that.